### DISCUSSION TITLE
Multiple subdomains from a single install?

### DISCUSSION THREAD

**Discussion by dzg** [YOURLS/YOURLS#3765] [Q&A]
My first day with YOURLS ... I have it installed under `domain.com` and working great.
But is there any way or plugin to be able to also create short URLs using `aaa.domain.com`, `bbb.domain.com` etc?
Thanks!

**Comment by dgw** [comment]
If you don't mind sharing the keyword namespace across all domains, you can redirect any additional domains you want to the main YOURLS instance using HTTP server configuration and it'll work.

Otherwise, YOURLS isn't built with multi-domain support. The underlying infrastructure for this feature exists in YOURLS v2, which is a near-complete rewrite, but we do not have an ETA on when that version might be ready to release.

**Reply by dzg** [reply]
I'll just use separate installs and wait for v2, thanks!

**Comment by dzg** [comment]
Sorry, can you also explain your suggestion of redirecting additional domains? I could use that in the meantime. Thanks!

**Reply by dgw** [reply]
Redirecting all traffic at `someother.domain` to `yourls.domain`. A request for e.g. `https://another.domain/aHytR` bounces to `https://yourls.domain/aHytR` where YOURLS takes over and does the actual redirection.

It's pretty easy in Apache, Nginx, Caddy, or just about any other HTTP server you might be using. The only downside, other than having to share the keyword namespace across all domains you set up this way (because there is only one YOURLS instance), is the added latency of a second DNS lookup and webserver round-trip.

**Reply by dzg** [reply]
I tried redirect (with wildcard) but keep getting 404 errors.

Do I need to edit `.htaccess` or `config.php`?

**Reply by dgw** [reply]
`.htaccess` on the other domain(s) if they use Apache. Don't add the redirection to the `.htaccess` for YOURLS
